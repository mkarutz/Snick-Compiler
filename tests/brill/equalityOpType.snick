proc main ()
    int x;
    float y;

    write x = y;
end
