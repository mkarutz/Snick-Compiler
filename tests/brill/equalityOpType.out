Failure: Arguments of equality operators must have the same type.
Compilation Failed
